website/content/en/preview/faq.md

```diff
@@ -118,6 +118,11 @@ If the instance type is unavailable for some reason, then fleet will move on to
 If you are using the spot capacity type, Karpenter uses the price-capacity-optimized allocation strategy. This tells fleet to find the instance type that EC2 has the most capacity for while also considering price. This allocation strategy will balance cost and decrease the probability of a spot interruption happening in the near term.
 See [Choose the appropriate allocation strategy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-fleet-allocation-strategy.html#ec2-fleet-allocation-use-cases) for information on fleet optimization.
 
+### How does Karpenter calculates the resource usage by Daemonsets while placing a pod?
```

Comment:
How does Karpenter calculate the resource usage of Daemonsets when simulating scheduling?

Reply:
Made the change to question.